README.md

```diff
@@ -15,6 +15,9 @@ Bevy is a refreshingly simple data-driven game engine built in Rust. It is free
 
 Bevy is still in the _very_ early stages of development. APIs can and will change (now is the time to make suggestions!). Important features are missing. Documentation is sparse. Please don't build any serious projects in Bevy unless you are prepared to be broken by API changes constantly.
 
+Bevy relies heavily on improvements in the Rust language and compiler.
```

Comment:
Best to have a more visible heading over this so it catches the attention of users or a bolded MSRV point before it.

Reply:
```suggestion
**MSRV:** Bevy relies heavily on improvements in the Rust language and compiler.
```